cli/plasmo/src/features/manifest-factory/base.ts

```diff
@@ -403,12 +403,16 @@ export abstract class PlasmoManifest<T extends ExtensionManifest = any> {
         )
       }
 
+      const isDev = process.env.NODE_ENV === "development"
+
       const contentScript = this.injectEnvToObj({
         matches: ["<all_urls>"],
         js: [
-          metadata?.config?.world === "MAIN"
-            ? scriptPath.split(scriptExt)[0]
-            : scriptPath
+          //!! I am not sure what this actually does??
+          // metadata?.config?.world === "MAIN" && isDev
+          //   ? scriptPath.split(scriptExt)[0]
+          //   : scriptPath
```

Comment:
This ensure we add those main world scripts into the CSP, but with the correct extension :-?

Reply:
Aa, I never had problems with that so I didn't realize. Thanks!